lib/vrl/tests/tests/rfcs/8381/check_property_on_variable_sized_array_of_objects.vrl

```diff
@@ -1,8 +1,7 @@
-# SKIP
 # result: true
 
-array =  [{ "a": 3 }, { "a": 2 }, { "a": 4 }]
+arr =  [{ "a": 3 }, { "a": 2 }, { "a": 4 }]
```

Comment:
Did we turn into a pirate here? :trollface:

Reply:
Haha, `array` is a reserved keyword. These tests were added in the relevant RFC, but they were marked as "skipped", so the syntax itself was never validated to work.